src/utils/problem.jl

```diff
@@ -3,15 +3,14 @@
     PolytopeComplement
 
 The complement to a given set. Note that with the exceptions of `Halfspace` and `EmptySet`,
- a `PolytopeComplement` is not a convex set.
-The.
+ a `PolytopeComplement` is not a convex set. Also note that `PolytopeComplement`s are open by definition.
```

Comment:
better to say: a `PolytopeComplement` may not be a convex set. 
For half space, the complement is convex.

Reply:
> For half space, the complement is convex.

This is what I was trying to say with:

>Note that with the exceptions of `Halfspace` and `EmptySet`, a `PolytopeComplement` is not a convex set.

but I guess it didn't get across as I intended

Reply:
Not sure if you use that, but the complement of an `EmptySet` is a `Universe`, so that would need to be added to the list.

Reply:
@schillic luckily then, we don't use it 😄. We don't actually use the `EmptySet` either, it just occurred to me as another exception to the convexity statement.